superset/key_value/api.py

```diff
@@ -48,65 +58,113 @@ class KeyValueRestApi(BaseApi, ABC):
     allow_browser_login = True
 
     def add_apispec_components(self, api_spec: APISpec) -> None:
-        api_spec.components.schema(
-            KeyValuePostSchema.__name__, schema=KeyValuePostSchema,
-        )
-        api_spec.components.schema(
-            KeyValuePutSchema.__name__, schema=KeyValuePutSchema,
-        )
+        try:
+            api_spec.components.schema(
```

Comment:
can we instead use `openapi_spec_component_schemas` for this?

Reply:
We can't because `openapi_spec_component_schemas` is defined in `BaseSupersetModelRestApi` and this endpoint does not have a database model. It inherits directly from `BaseApi`.